flask_jwt_router/_extensions.py

```diff
@@ -33,23 +35,27 @@ def __init__(self,
 
```

Comment:
Improved the tests for `extensions` .. should only be testing the actual `extensions` class...